src/Mod/Part/AttachmentEditor/TaskAttachmentEditor.py

```diff
@@ -523,20 +523,21 @@ def updateListOfModes(self):
                 for refstr in mi['ReferenceCombinations']:
                     refstr_userfriendly = [self.attacher.getRefTypeInfo(t)['UserFriendlyName'] for t in refstr]
                     cmb.append(u", ".join(refstr_userfriendly))
-                
+
                 tip = mi['BriefDocu']
                 if (m != 'Deactivated'):
-                    tip += _translate('AttachmentEditor', "\n\nReference combinations:\n", None) + u"\n".join(cmb)
-                
+                    tip += QString::fromLatin1("\n\n")
+                    tip += _translate('AttachmentEditor', "Reference combinations:", None) + u"\n\n".join(cmb)
```

Comment:
Looks like there is an issue here: https://travis-ci.org/FreeCAD/FreeCAD/jobs/483951642#L8577

Reply:
The error is not very descriptive. Looks like this is failing in a test, not in the compilation directly. Any insight would help, thanks.

Reply:
testAll (TestPythonSyntax.PythonSyntaxTestCase) ... ERROR

Is the python self-test-script that fails atleast. Do not know enough python yet to help much

Reply:
Fixed in https://github.com/FreeCAD/FreeCAD/pull/1912/commits/26f6df1967a8d804d5c85b586d47a626f7fde24e